pyproject.toml

```diff
@@ -1,3 +1,35 @@
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools >= 36.6"]
+requires = ["setuptools >= 61.0"]
+
+[project]
+name = "colored"
+version = "0.4.0"
+description = "Automatically color uncaught exception tracebacks"
+authors = [
+    { name = "Anton Backer", email = "olegov@gmail.com" },
+]
+readme = "README.rst"
+license = { text = "ISC" }
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "License :: OSI Approved :: ISC License (ISCL)",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.7",
```

Comment:
If the mission is to translate without changing then 2.7 + 3.3 would be that translation. Else, if the mission is to also get onto supported Python version ground then 3.8 would be the minimum, [3.7 is already end of life](https://endoflife.date/python). What do you think?

Reply:
Setuptools v61 only supports Python 3.7+, but I think here is not the place to work about that: I'll leave the classifiers unchanged.